pages/INSTALL.md

```diff
@@ -0,0 +1,1182 @@
+
+
+
+
+
+
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
What is the purpose of this file?

Reply:
Not sure. I do not recall creating it. I'll delete it.